telco-hub/configuration/reference-crs/required/gitops/ztp-installation/app-project.yaml

```diff
@@ -29,16 +27,12 @@ spec:
     kind: NMStateConfig
   - group: 'extensions.hive.openshift.io'
     kind: AgentClusterInstall
-  - group: 'extensions.hive.openshift.io'
-    kind: ImageClusterInstall
```

Comment:
I think we want to keep these? Maybe they need to be updated in the ran reference also so that they are kept here? (same for DataImage and sync-wave)